packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -484,14 +484,14 @@ export abstract class SharedObject<TEvent extends ISharedObjectEvents = ISharedO
      * {@inheritDoc (ISharedObject:interface).getAttachSummary}
      */
     public getAttachSummary(fullTree: boolean = false, trackState: boolean = false): ISummaryTreeWithStats {
```

Comment:
I'm not sure IFluidDataStoreChannel is relevant here. Did you mean IChannel?

Reply:
No, I was looking at other things with this pattern to understand when fullTree might be used when summarizing. Do we want to potentially unify IChannel and IFluidDataStoreChannel? or am I getting that confused with something else?

Reply:
Well, yes I really want that, but it will take months of work, I think, if you are asking about whole interfaces :)
If you refer only to summarization, then I think it also big, as 2-phase async summary will take time to implement at higher layers.
As for a flag only, I believe it’s present on both layers and has to be there, as we do want to support incremental summaries, even though most of our DDSs do not support it